### PR TITLE
Signature formats sample and public key verification

### PR DESCRIPTION
**Pull request by bolzon**
How about now?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,23 @@ var derSign = signature.toDER();
...
 console.log(key.verify(msg, derSign));
+
+// CHECK WITH NO PRIVATE KEY
+
+// Public key as '04 + x + y'
+var pub = '04bb1fa3...';
+
+// Signature MUST be either hex-string, DER-encoded buffer or an object with two properties (r and s), which should be hex strings
+
+var signature = 'b102ac...'; // hex-string
```
Comment (indutny) [inline README.md]: `hex-string of DER-encoded signature`

File: README.md [hunk 2]
```diff
@@ -62,6 +62,23 @@ var derSign = signature.toDER();
 
 // Verify signature
 console.log(key.verify(msg, derSign));
+
+// CHECK WITH NO PRIVATE KEY
+
+// Public key as '04 + x + y'
+var pub = '04bb1fa3...';
+
+// Signature MUST be either hex-string, DER-encoded buffer or an object with two properties (r and s), which should be hex strings
```
Comment (indutny) [inline README.md]: 80 column limit, please.

**Comment by indutny** [comment]
Looks terrific! Thank you so much. This PR has only two minor nits, and otherwise is ready to go!

**Comment by bolzon** [comment]
Welcome.
Well, the case about hex-string is sort of obvious for me, but anyway.
About 80-column style it's okay, I will redo this.

**Comment by indutny** [comment]
Thanks a lot!

**Comment by bolzon** [comment]
Done, thank you ;)

**Comment by indutny** [comment]
Fantastic!

**Comment by indutny** [comment]
Landed in 58b2ee15346315d6f8c16313005f9421c2599df3 , thank you!

**Pull request closed without merging**